Makefile

```diff
@@ -208,6 +208,10 @@ $(BUILDDIR)/teleport: ensure-webassets bpf-bytecode rdpclient
 $(BUILDDIR)/tsh:
 	GOOS=$(OS) GOARCH=$(ARCH) $(CGOFLAG_TSH) go build -tags "$(PAM_TAG) $(FIPS_TAG)" -o $(BUILDDIR)/tsh $(BUILDFLAGS) ./tool/tsh
 
+.PHONY: $(BUILDDIR)/tbot
+$(BUILDDIR)/tbot:
+	GOOS=$(OS) GOARCH=$(ARCH) $(CGOFLAG) go build -tags "$(PAM_TAG) $(FIPS_TAG)" -o $(BUILDDIR)/tbot $(BUILDFLAGS) ./tool/tbot
```

Comment:
Do you get different behavior if you omit the PAM/FIPS tags? If not, can we do without them?

Reply:
I was about to say "we can just drop them" but I suppose it's possible the FIPS tag is honored in some of the teleport libraries we use. PAM is almost certainly unnecessary but I included it since `tsh` does for some reason as well.

Reply:
I've removed the PAM tag. I'm wary of touching the FIPS tag since I'm not sure what it touches. My gut feeling is that if `tsh` cares, we should too.